### DISCUSSION TITLE
updating merchant_release_date with patchListingsItem

### DISCUSSION THREAD

**Discussion by PhenomEY** [amzn/selling-partner-api-models#3351] [General]
Hello 😊

im trying to update the merchant_release_date with the patchListingsItem Operation. I cant find any documentation on the required value for the attribute, can someone help me out?

**Comment by chapmanjw** [comment]
@PhenomEY You can use the Product Type Definitions API to get the JSON Schemas describing the attributes and their requirements for a given product type: https://developer-docs.amazon.com/sp-api/docs/product-type-api-use-case-guide

A `patchListingsItem` request for this attribute would look something like this:

```
{
  "productType":"PRODUCT",
  "patches":[
    {
      "op":"replace",
      "path":"/attributes/merchant_release_date",
      "value":[
        {
          "value": "2023-01-01",
          "marketplace_id": "ATVPDKIKX0DER"
        }
      ]
    }
  ]
}
```

**Comment by l20DfX35JnKBfRn** [comment]
This helped me with below:

If you want to update `merchant_release_date` for an item with JSON_LISTING_FEEDS:
```json
{
      "messageId": 1,
      "sku": "${SKU}",
      "operationType": "PATCH",
      "productType": "${PRODUCT_TYPE}",
      "patches": [
        {
          "op": "replace",
          "path": "/attributes/merchant_release_date",
          "value": [
            {
              "value": "${YYYY_MM_DD}"
            }
          ]
        }
      ]
    }
```

https://developer-docs.amazon.com/sp-api/docs/feed-type-values
